lg_common/src/lg_common/uscs_service.py

```diff
@@ -42,7 +42,7 @@ def __init__(self,
             rospy.loginfo("Enabling on_offline_state handling")
             self.on_offline_state = self._grab_scene(on_offline_state_scene_url)
         else:
-            rospy.loginfo("Disabling on_offline_state handling")
+            rospy.loginfo("Will use initial state for on_offline_state")
```

Comment:
That's a good loginfo. I cant imagine a state where dont want to revert to default state when going offline.